go.mod

```diff
@@ -2,11 +2,12 @@ module github.com/OctopusDeploy/cli
 
 go 1.21
 
+
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/MakeNowJust/heredoc/v2 v2.0.1
 	github.com/OctopusDeploy/go-octodiff v1.0.0
-	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.59.0
+	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.62.3-0.20241202024636-075a4f06c227
```

Comment:
Use https://github.com/OctopusDeploy/go-octopusdeploy/pull/288 for now.

Will update with the correct version once go client is merged after octopus server changes.